pymc3/tuning/starting.py

```diff
@@ -16,9 +17,8 @@
 
 __all__ = ['find_MAP']
 
-
 def find_MAP(start=None, vars=None, fmin=None,
-             return_raw=False,model=None, *args, **kwargs):
+             return_raw=False, model=None, live_disp=False, callback=None, *args, **kwargs):
```

Comment:
add docs for `live_disp`, also, it doesn't look like an argument for `callback` is respected below, so it could just be initialized as `None`, right?